queries.sql

```diff
@@ -9,3 +9,100 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+-- Begin a transaction
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+-- Veryfying that change was made in the database
+SELECT * FROM animals;
+
+--Rollback the transaction
+ROLLBACK;
+
+--Verify that the transaction was successful
+SELECT * FROM animals;
+
+--Update the species colunm
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
```

Comment:
* Kindly write queries for these missing transactions as its a [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20changes,persist%20after%20commit.)
```
 Verify that changes were made.
Commit the transaction.
Verify that changes persist after the commit.
```

Reply:
@Davidosky007 
Thanks for reminding me

Reply:
This is resolved. Thanks